app/src/editor/shared/components/modules/Table.jsx

```diff
@@ -52,7 +52,7 @@ const parseMessage = (d, options) => (state) => {
     } else if (d.nm) {
         // new map
         newState.rows = []
-        Object.keys(d.dm).forEach((key) => {
+        Object.keys(d.nm).forEach((key) => {
```

Comment:
I assumed this is a typo, please verify if not. It gave an error and this fixed it.

Reply:
typo 👍